Packs/MicrosoftTeams/Integrations/MicrosoftTeams/MicrosoftTeams.py

```diff
@@ -108,6 +108,46 @@ class FormType(Enum):  # Used for 'send-message', and by the MicrosoftTeamsAsk s
 TOKEN_EXPIRED_ERROR_CODES = {50173, 700082, }  # See: https://login.microsoftonline.com/error?code=
 REGEX_SEARCH_ERROR_DESC = r"^[^:]*:\s(?P<desc>.*?\.)"
 
+COMMANDS_REQUIRED_PERMISSIONS: dict[str, list] = {
+    # Note: at the moment, the required permission names between credentials and auth code are the same.
+    # Credentials require Application permissions while auth code requires delegated permissions
+    'send-notification': ['GroupMember.Read.All', 'Channel.ReadBasic.All'],
```

Comment:
Please make all the permission constant for example GROUP_MEMBER = 'GroupMember', READ = 'Read', ALL = 'All' etc..